_docs/tasks/traffic-management/ingress.md

```diff
@@ -269,6 +269,344 @@ rules.
    curl -I -k https://$INGRESS_HOST/status/200
    ```
 
+## Configuring RBAC for ingress key/cert
+There are service accounts which can access this ingress key/cert, and the ingress key/cert is not 
+encrypted. This leads to risks of leaking key/cert. We can set up Role-Based Access Control ("RBAC") 
+to protect the ingress key/cert. [istio/install/kubernetes/istio.yaml](https://github.com/istio/istio/blob/master/install/kubernetes/istio.yaml) 
+defines some ClusterRoles which grant access to all secret resources. For example, 
+istio-pilot-istio-system and istio-mixer-istio-system allow reading secret resources, and 
+istio-ca-istio-system allows reading, creating and updating secret resources.
+[istio/install/kubernetes/istio.yaml](https://github.com/istio/istio/blob/master/install/kubernetes/istio.yaml) 
+also defines ClusterRoleBindings which bind these ClusterRoles to service accounts, such as 
+istio-pilot-service-account, istio-mixer-service-account, istio-ca-service-account and 
+istio-ingress-service-account. We need to update or replace these RBAC set up to only allow 
+istio-ingress-service-account to access ingress key/cert.       
+
+1. Update RBAC set up for istio-pilot-service-account and istio-mixer-istio-service-account.
```

Comment:
Please help to keep the new content succinct and only display things of interest.

Reply:
I have made the content shorter now.